build/pipelines/templates/helix-runtests-job.yml

```diff
@@ -0,0 +1,131 @@
+parameters:
+  name: 'RunTestsInHelix'
+  dependsOn: ''
+  condition: ''
+  testSuite: ''
+  # If a Pipeline runs this template more than once, this parameter should be unique per build flavor to differentiate the
+  # the different test runs:
+  helixType: 'test/devtest' 
+  artifactName: 'drop'
+  maxParallel: 4
+  rerunPassesRequiredToAvoidFailure: 5
+  taefQuery: ''
+  # if 'useBuildOutputFromBuildId' is set, we will default to using a build from this pipeline:
+  useBuildOutputFromPipeline: $(System.DefinitionId)
+  matrix: 
+    # Release_x86:
+    #   buildPlatform: 'x86'
+    #   buildConfiguration: 'release'
+    #   openHelixTargetQueues: 'windows.10.amd64.client19h1.open.xaml'
+    #   closedHelixTargetQueues: 'windows.10.amd64.client19h1.xaml'
+    Release_x64:
+      buildPlatform: 'x64'
+      buildConfiguration: 'release'
+      openHelixTargetQueues: 'windows.10.amd64.client19h1.open.xaml'
+      closedHelixTargetQueues: 'windows.10.amd64.client19h1.xaml'
+
+jobs:
+- job: ${{ parameters.name }}
+  dependsOn: ${{ parameters.dependsOn }}
+  condition: ${{ parameters.condition }}
+  pool:
+    vmImage: 'windows-2019'
+  timeoutInMinutes: 120
+  strategy:
+    maxParallel: ${{ parameters.maxParallel }}
+    matrix: ${{ parameters.matrix }}
+  variables:
+    artifactsDir: $(Build.SourcesDirectory)\Artifacts
+    taefPath: $(Build.SourcesDirectory)\build\Helix\packages\taef.redist.wlk.10.51.200127004\build\Binaries\$(buildPlatform)
```

Comment:
old taef path

Reply:
Good call. We technically have both, but good call.